Pizzeria/Pizzeria/Employee/Administrator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Pizzeria
+{
+    public class Administrator : Employee
+    {
+        private List<Employee> _employees = new List<Employee>();
+        public Administrator(string name)
+        {
+            Name = name;
+        }
+        public void AddEmployee(Employee employee)
+        {
+            _employees.Add(employee);
+        }
+        public void CreateSalary()
+        {
+            foreach (var employee in _employees)
+            {
+                if(employee.WorkExperience<=2)
+                {
+                    employee.Salary = employee.WorkExperience * 1000;
+                }
+                else if(employee.WorkExperience>2 && employee.WorkExperience<=5)
+                {
+                    employee.Salary = employee.WorkExperience * 2000;
+                }
+                else
+                {
+                    employee.Salary = employee.WorkExperience * 3000;
+                }
+                
+            }
+        }
```

Comment:
I do not expect to see this in Administrator class. How does the administrator gets the salary?
This method should be somewhere esle but not here